my-app/package.json

```diff
@@ -3,19 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "lodash": "^4.17.5",
-    "prop-types": "^15.6.1",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-redux": "^5.0.7",
-    "react-router-dom": "^4.2.2",
-    "react-scripts": "1.1.1",
+    "lodash": "^4.17.21",
+    "prop-types": "^15.7.2",
```

Comment:
You don't need to install `prop-types` manually. It's loaded with `react` after v.16